ios/fastlane/Appfile

```diff
@@ -0,0 +1,4 @@
+app_identifier  "io.parity.NativeSigner"
+apple_id        "krzysztof@parity.io"
```

Comment:
All relevant actions are done through API key, this is only used if there is a need to refresh Provisioning Profiles